### PR TITLE
fix(lsp): divide hcl into seperate languages

### PR DESCRIPTION
**Pull request by hown3d**
Terraform-ls uses two seperate language-ids for terraform and terraform-vars files.

Read more about it [here](https://github.com/hashicorp/terraform-ls/blob/main/docs/language-clients.md#language-ids)

### CODE REVIEW

File: languages.toml [hunk 1]
```diff
@@ -1038,17 +1038,31 @@ source = { git = "https://github.com/fwcd/tree-sitter-kotlin", rev = "a4f71eb9b8
...
+file-types = ["tfvars"]
+roots = []
+comment-token = "#"
+indent = { tab-width = 2, unit = "  " }
+language-server = { command = "terraform-ls", args = ["serve"], language-id = "terraform-vars" }
+auto-format = true
+
+[[grammar]]
+name = "tfvars"
+source = { git = "https://github.com/MichaHoffmann/tree-sitter-hcl", rev = "3cb7fc28247efbcb2973b97e71c78838ad98a583" }
```
Comment (the-mikedavis) [inline languages.toml:1064]:
```suggestion
grammar = "hcl"
```

you can avoid duplicating re-used grammars with the `grammar` key

**Review by the-mikedavis — CHANGES_REQUESTED** [review]
Just a little duplication nit, otherwise I think this is good 👍

File: languages.toml (hunk 1 above)
Comment (hown3d) [reply languages.toml:1064]: Ah okay, good to know!

**Review by the-mikedavis — APPROVED** [review]
looks good, thanks!

**Pull request merged**